README.md

```diff
@@ -162,6 +162,22 @@ SET_URL=http://localhost:8983/solr/orangelight-core-development BIB=123456 rake
 ## Alma Webhooks
 see [[webhook_monitor/README.md]]
 
+## MARC Files
+If you have MARC file you can import it to Solr via Traject with the following commands:
+
+```
+FILE=/path/to/marc/file/filename.xml
+SOLR_URL=http://localhost:8983/solr/puldata
```

Comment:
Great @hectorcorrea . Thanks! Can you add a general solr name? `SOLR_URL=http://localhost:8983/local-solr-url`

Reply:
Good idea. Done.